src/models/profile.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+import mongoose from 'mongoose';
+import User from '../auth/user';
+import StatBlock from '../models/statBlock';
+
+const Schema = mongoose.Schema;
+
+const profileSchema = Schema({
+  user: { type: Schema.Types.ObjectId, ref: 'users' },
+  firstName: { type: String, required: true },
+  lastName: { type: String, required: true },
+  bio: { type: String, default: `Hello!` },
+  email: { type: String },
+  role: { type: String },
+
+});
+
+profileSchema.pre('save', function (next) {
+  let profileId = this._id;
+  let role = this.role;
+  let userId = this.user;
+  let statBlock = new StatBlock({ user: this.user });
```

Comment:
This should be placed in a try/catch block in the event the StatBlock is unable to be created off the data you pass in, or if the save function below encounters an error.